packages/DraftEditor/package.json

```diff
@@ -18,9 +18,16 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@blaze-react/file-upload": "^0.3.32",
     "@blaze-react/utils": "^0.3.31",
+    "@blaze-react/modal": "^0.3.32",
+    "@blaze-react/input": "^0.3.31",
     "@types/draft-js": "^0.10.34",
-    "draft-js": "^0.11.0"
+    "draft-js": "^0.11.0",
```

Comment:
@oscar-leon @Arcoders I think this kind of dependencies should be freeze in order to avoid have crashes in production. We should keep an eye on this dependencies and update it when will be necessary.